reference/7.2/PSReadLine/About/about_PSReadLine.md

```diff
@@ -1731,7 +1728,63 @@ example, the user is reading help to decide which parameter to use.
 Beginning in PSReadLine 2.2.0-beta4, **OnIdle** behavior changed to signal the event only if there
 is a `ReadKey()` timeout and the current editing buffer is empty.
 
-### Feedback & Contributing To PSReadLine
+### PSReadLine release history
+
+There have been many updates to PSReadLine since the version that ships in
+Windows PowerShell 5.1.
+
+- PowerShell 7.3-preview.5 ships with PSReadLine 2.2.5
+- PowerShell 7.2.5 ships with PSReadLine 2.1.0
+- PowerShell 7.0.11 ships with PSReadLine 2.0.4
+- PowerShell 5.1 ships with PSReadLine 2.0.0
+
+For a full list of changes, see the PSReadLine
+[ChangeLog](https://github.com/PowerShell/PSReadLine/blob/master/PSReadLine/Changes.txt).
+
+- **PSReadLine 2.2.6**
+
+  In this release, the Predictive IntelliSense feature is enabled by default
+  depending on the following conditions:
+
+  - If Virtual Terminal (VT) is supported and PSReadLine running in PowerShell
+    7.2 or higher, **PredictionSource** is set to `HistoryAndPlugin`
+  - If VT is supported and PSReadLine running in PowerShell prior to 7.2,
+    **PredictionSource** is set to `History`
+  - If VT is not supported, **PredictionSource** is set to `None`
+
+- **PSReadLine 2.2.5**
+
+  This release rolls up the following enhancements added since the 2.1.0
+  release:
+
+  - PSReadLine added two new predictive IntelliSense features:
+    - Added the **PredictionViewStyle** parameter to allow for the selection of
+      the new `ListView`.
+    - Connected PSReadLine to the `CommandPrediction` APIs introduced in
+      PowerShell 7.1 to allow a user can import a predictor module that can
```

Comment:
Should be 7.2

Reply:
This change was introduced in 7.1.

Reply:
This change was done in 7.2. You can import predictor module only in 7.2